src/models/network-v2.ts

```diff
@@ -100,25 +100,12 @@ export class Network_v2 extends Model<Network_v2Methods> implements Deployable {
     return this.callTx(this.contract.methods.treasuryInfo());
   }
 
-  /**
-   * Calculate the values that is distributed when a bounty is closed
-   * @param bountyAmount bounty's value
-   * @param proposalPercents an array with the percentuals of a proposal
-   */
-  async calculateDistributedAmounts(bountyAmount: number, proposalPercents: number[]) {
-    const treasury = await this.treasuryInfo();
-
-    const treasuryAmount = treasury['0'] === nativeZeroAddress ? 0 :  ((bountyAmount / 100) * (treasury['1'] / TenK));
-    const mergerAmount = bountyAmount / 100 * await this.mergeCreatorFeeShare();
-    const proposerAmount = (bountyAmount - mergerAmount) / 100 * await this.proposerFeeShare();
-    const amount = bountyAmount - treasuryAmount - mergerAmount - proposerAmount;
-
-    return {
-      treasuryAmount,
-      mergerAmount,
-      proposerAmount,
-      proposals: proposalPercents.map(percent => (amount / 100 * percent))
-    };
+  async claimGovernor() {
```

Comment:
If you're going to add these functions, let me introduce you to the awesomeness that is `Governed` and [how it is used](https://github.com/taikai/dappkit/blob/d9268a73e6086813ac5b936be65e4f88f3acfd8d/src/models/network-v2.ts#L66)